libs/coin-modules/coin-evm/src/api/node/ledger.ts

```diff
@@ -258,21 +258,22 @@ export const getFeeData: NodeApi["getFeeData"] = async (currency, transaction) =
 export const broadcastTransaction: NodeApi["broadcastTransaction"] = async (
   currency,
   signedTxHex,
+  broadcastConfig,
 ) => {
   const config = getCoinConfig(currency).info;
   const { node } = config || /* istanbul ignore next */ {};
   if (!isLedgerNodeConfig(node)) {
     throw new LedgerNodeUsedIncorrectly();
   }
 
+  const mevParam = broadcastConfig?.mevProtected ? "?mevProtected=true" : "";
   const { result: hash } = await fetchWithRetries<{
     result: string;
   }>({
     method: "POST",
-    url: `${getEnv("EXPLORER")}/blockchain/v4/${node.explorerId}/tx/send`,
+    url: `${getEnv("EXPLORER")}/blockchain/v4/${node.explorerId}/tx/send${mevParam}`,
     data: { tx: signedTxHex },
   });
```

Comment:
Try not using string concatenation and rather use the `params` parameter of `axios`:

```typescript
  const { result: hash } = await fetchWithRetries<{
    result: string;
  }>({
    method: "POST",
    url: `${getEnv("EXPLORER")}/blockchain/v4/${node.explorerId}/tx/send`,
    data: { tx: signedTxHex },
    params: {
        mevProtected: Boolean(broadcastConfig?.mevProtected)
    }
  });
```

Reply:
https://github.com/LedgerHQ/ledger-live/pull/8375/commits/322893ec2bbb39ceaa8fe8d9109c7a97c7ab3f5d